department-of-veterans-affairs/abd-vro
# MCP-2501 PDF Generator CSS File Setup

## What was the problem?

All the styling was inline. It makes the template harder to read and also troubleshoot if it requires updates in the future

Associated tickets or Slack threads:

- [MCP-2501](https://amida.atlassian.net/browse/MCP-2501)

## How does this fix it?[^1]

CSS styling was moved to a separate `base.css` file. Not everything was moved, column widths were left as inline styles for now

## How to test this PR
- Generate a PDF based off the `develop` branch and then generate the same PDF off of this branch. Both should match


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
